flow-decomposition/src/test/java/com/powsybl/flow_decomposition/FlowDecompositionObserverTest.java

```diff
@@ -295,12 +317,12 @@ void testNStateN1AndN2PostContingencyState() {
 
         // Checking AC flows
         for (var contingencyId : List.of(BASE_CASE, contingencyId1, contingencyId2)) {
-            assertEquals(allBranches, report.acFlows.forContingency(contingencyId).keySet());
+            assertEquals(allBranches, report.acFlowsTerminal1.forContingency(contingencyId).keySet());
         }
 
         // Checking DC flows
         for (var contingencyId : List.of(BASE_CASE, contingencyId1, contingencyId2)) {
-            assertEquals(allBranches, report.dcFlows.forContingency(contingencyId).keySet());
+            assertEquals(allBranches, report.acFlowsTerminal1.forContingency(contingencyId).keySet());
```

Comment:
`dcFlows` instead of `acFlowsTerminal1` no ?

Reply:
Ops, thanks